index.js

```diff
@@ -190,3 +190,130 @@ const titleCase = str => {
     })
     .join(" ");
 };
+
+//Task 26
+
+const getAverage = marks => {
+  const allValues = marks.reduce((accum, curVal) => accum + curVal);
+  return Math.round(allValues / marks.length);
+};
+
+//Task 27
+
+class Human {
+  constructor() {}
+}
+class Man extends Human {
+  sex = "male";
+  constructor(name) {
+    super();
+    this.name = name;
+  }
+}
+class Woman extends Human {
+  sex = "female";
+  constructor(name) {
+    super();
+    this.name = name;
+  }
+}
+
+class God {
+  constructor() {}
+  static create(manName, womanName) {
+    let man = new Man();
+    man.name = manName;
```

Comment:
you can pass name parameter into Class: `new Man(manName)`, this way you pass parameters into Class constructor